portability-gateway/src/main/java/org/dataportabilityproject/gateway/ApiMain.java

```diff
@@ -89,6 +91,9 @@ public void initializeHttps(
 
     ApiExtensionContext extensionContext = new ApiExtensionContext(typeManager, settingsExtension);
 
+    extensionContext.registerService(HttpTransport.class, new NetHttpTransport());
```

Comment:
Didn't this already exist?

Reply:
Nope, they were null when they were retrieved from the context later. It existed in the worker, I copied from there.